LibOS/shim/src/fs/proc/info.c

```diff
@@ -29,6 +31,34 @@ int proc_meminfo_load(struct shim_dentry* dent, char** out_data, size_t* out_siz
             "MemFree:       %8lu kB\n",
             DkMemoryAvailableQuota() / 1024,
         },
+        {
+            "MemAvailable:  %8lu kB\n",
+            DkMemoryAvailableQuota() / 1024,
+        },
+        {
+            "Buffers:       %8lu kB\n",
+            /*dummy value=*/0,
+        },
+        {
+            "Cached:        %8lu kB\n",
+            /*dummy value=*/0,
+        },
+        {
+            "Shmem:         %8lu kB\n",
+            /*dummy value=*/0,
+        },
+        {
+            "Active:        %8lu kB\n",
+            /*dummy value=*/0,
+        },
+        {
+            "Inactive:      %8lu kB\n",
+            /*dummy value=*/0,
+        },
+        {
+            "Slab:          %8lu kB\n",
+            /*dummy value=*/0,
+        },
```

Comment:
Might need more if we also want to satisfy Rust workloads.
See here for those expecting fields:
https://docs.rs/procfs/latest/src/procfs/meminfo.rs.html#323-382